BER-FT-0321-Svjetlana Rados

SvjetlanaR opened the pull request.

wax-bear reviewed (commented):
Great stuff Svjetlana!

Is there a particular reason for using "<!section 1/3!>" this type of element/statement? If not, I would not include them in the HTML.

Seeing your website, I notice a few matters that can be improved to increase the resemblance between your clone and the original:

1. Adjusting font sizes and styles (The tile, the nav bar items, etc.)
2. Working on the position of elements, most notably the search bar

Please reach out if you have a question.

Best,

Thomas (TA WDFT)

SvjetlanaR commented:
Thanks Thomas!

I added these comments to have the HTML more structured, but I removed them. 

Yes, I will change the style of the search bar.

Warm regards,
Svjetlana
